server/index.js

```diff
@@ -0,0 +1,31 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var songData = require('../songData');
+var path = require('path');
+var app = express();
+var port = 3000;
+
+app.use(express.static(path.join(__dirname + '../client')));
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
+
+// INCLUDE "try" AND "catch" IN THE ASYNC AWAIT BELOW
+
+app.get('/songdata/:id', async (req, res) => {
+  try {
+    var id = req.params.id;
+    if (id > 100 || id < 0) {
+      res.end('SONG DOES NOT EXIST');
+    } else {
+      var findSongByID = await songData.findSong(id)
```

Comment:
What if there is an error from songData? this is what should be in the try block